conf/messages.en

```diff
@@ -1292,18 +1292,18 @@ messages__schemeTaskList__sectionEstablishersCompany_add_address = Add {0}’s a
 messages__schemeTaskList__sectionEstablishersCompany_add_contact = Add {0}’s contact details
 messages__schemeTaskList__sectionEstablishersCompany_add_directors = Add {0}’s directors
 
```

Comment:
Duplicate messages. Reuse the ones for establisherCompany by giving generic names

Reply:
done